### PR TITLE
Reuse ovs-cni privileged POD

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: tests/manifests/network/ovs-vlan-net-vxlan.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-vlan-net-vxlan
+  namespace: cnv-network-ns
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
```
Comment (ILpinto) [inline tests/manifests/network/ovs-vlan-net-vxlan.yml]: maybe keep version outside yamls file and use oc process

File: tests/network/conftest.py [hunk 2]
```diff
@@ -27,3 +28,7 @@ def fin():
     ns.create(wait=True)
     ns.wait_for_status(status=types.ACTIVE)
     ns.work_on()
+    pods = Pod().list(get_names=True)
+    pods = [i for i in pods if i in "ovs-cni"]
+
+    "oc exec -it ovs-cni-amd64-fzrrv -n kube-system -c ovs-cni-marker bash"
```
Comment (ILpinto) [inline tests/network/conftest.py]: ovs-cni-amd64-fzrrv is not the pod name ? it will change.

File: tests/network/connectivity/fixtures.py [hunk 3]
```diff
@@ -66,19 +61,19 @@ def fin():
...
+    for idx, pod in enumerate(get_ovs_cni_pods()):
+        pod_object = Pod(name=pod, namespace=config.KUBE_SYSTEM_NS)
+        pod_container = config.OVS_CNI_CONTAINER
         active_node_nics[pod] = []
         assert pod_object.wait_for_status(status=types.RUNNING)
         err, nics = pod_object.run_command(command=config.GET_NICS_CMD, container=pod_container)
         assert err
         nics = nics.splitlines()
-        err, default_gw = pod_object.run_command(command=config.GET_DEFAULT_GW_CMD, container=pod_container)
+        err, default_gw = pod_object.run_command(command="ip route show default", container=pod_container)
```
Comment (ILpinto) [inline tests/network/connectivity/fixtures.py]: better to add commands to config.

File: tests/network/connectivity/fixtures.py [hunk 4]
```diff
@@ -66,19 +61,19 @@ def fin():
...
         assert err
         for nic in nics:
             err, nic_state = pod_object.run_command(
-                command=config.GET_NIC_STATE_CMD.format(nic=nic), container=pod_container
+                command=f"cat /sys/class/net/{nic}/operstate", container=pod_container
```
Comment (ILpinto) [inline tests/network/connectivity/fixtures.py]: same

File: tests/network/connectivity/fixtures.py [hunk 5]
```diff
@@ -164,28 +159,23 @@ def fin():
...
-                container=pod_container
+                command=f"{config.OVS_VSCTL_ADD_PORT} {bond_bridge} {bond_name}", container=pod_container
             )[0]
 
     for vm in vms:
         vm_object = VirtualMachine(name=vm, namespace=config.NETWORK_NS)
-        json_out = utils.get_json_from_template(
-            file_=config.VM_YAML_TEMPLATE, NAME=vm, MULTUS_NETWORK="ovs-vlan-net"
-        )
+        network = "ovs-vlan-net" if config.REAL_NICS_ENV else "ovs-vlan-net-vxlan"
```
Comment (ILpinto) [inline tests/network/connectivity/fixtures.py]: same for: "ovs-vlan-net" , "ovs-vlan-net-vxlan"
use config

File: tests/network/connectivity/fixtures.py (hunk 3 above)
Comment (myakove) [reply tests/network/connectivity/fixtures.py]: only used once so I will keep it here also it's more readability like this.

File: tests/network/connectivity/fixtures.py (hunk 5 above)
Comment (myakove) [reply tests/network/connectivity/fixtures.py]: used only once

File: tests/manifests/network/ovs-vlan-net-vxlan.yml (hunk 1 above)
Comment (myakove) [reply tests/manifests/network/ovs-vlan-net-vxlan.yml]: I think oc process need a template, right?

Comment (ILpinto) [reply tests/manifests/network/ovs-vlan-net-vxlan.yml]: yes, Or you can replace with format and use oc create.

Comment (ILpinto) [reply tests/manifests/network/ovs-vlan-net-vxlan.yml]: you want to use constant version ?

Comment (myakove) [reply tests/manifests/network/ovs-vlan-net-vxlan.yml]: I guess, but not in this patch.

**Pull request merged**